### ISSUE TITLE
Add Bing redirect

### ISSUE THREAD

**Issue by BigBadE** [SimonBrazell/privacy-redirect#208]
EdgeDeflector is a popular tool to redirect Windows searches from Edge to their browser of choice. The problem is, EdgeDeflector doesn't change the search engine also, just the browser.

Bing has also [been hit with a massive cyberattack](https://www.wizcase.com/blog/bing-leak-research/), along with known violations of user privacy. Therefore, Bing should be added as a redirect as well as Google.

**Comment by SimonBrazell** [comment]
Good idea, it does describe it as "Search Engine" redirects which sounds generic but it in fact just redirects Google, perhaps I can add an advanced option to target other search engines.
